src/javascript/app/common/content_visibility.js

```diff
@@ -47,16 +46,26 @@ const mt_company_rule   = 'mtcompany';
 
 const ContentVisibility = (() => {
     const init = () => {
-        if (Client.isLoggedIn()) {
+        const client_is_logged_in = Client.isLoggedIn();
+        if (client_is_logged_in) {
             BinarySocket.wait('authorize', 'landing_company').then(() => {
+                const is_virtual      = State.getResponse('authorize.is_virtual');
+                const landing_company = is_virtual ?
+                    State.getResponse('landing_company.gaming_company.shortcode') ||
+                    State.getResponse('landing_company.financial_company.shortcode') :
```

Comment:
can you put another bracket around lines 54 and 55 so it's obvious that they're together?